nilearn/tests/test_masking.py

```diff
@@ -16,11 +16,10 @@
 from nilearn.masking import (compute_epi_mask, compute_multi_epi_mask,
                              compute_background_mask, unmask, _unmask_3d,
                              _unmask_4d, intersect_masks, MaskWarning)
+from nilearn._utils.testing import (write_tmp_imgs, assert_raises_regex,
+                                    assert_warns_regex)
```

Comment:
Thanks for the catch!  This made me realize, just now: one downside of using parentheses in imports is that sublime text no longer indicates which imports are being used.